internal/validations/observability/outputs/otlp.go

```diff
@@ -0,0 +1,17 @@
+package outputs
+
+import (
+	internalcontext "github.com/openshift/cluster-logging-operator/internal/api/context"
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+	"strings"
+)
+
+func ValidateOtlpAnnotation(context internalcontext.ForwarderContext) (results []string) {
```

Comment:
This needs a unit test to support the changes

Reply:
In progress.